Scripts/NaturalDisasters/DisasterPropertiesSandstorm.cs

```diff
@@ -6,7 +6,7 @@ public class DisasterPropertiesSandstorm : DisasterProperties
 	
 	public DisasterPropertiesSandstorm() : base() {
 		WarningTime = 10;
-		Likelihood = 0.005;
+		Likelihood = 0.01;
```

Comment:
Do we really want to increase the likelihood? I found them pretty annoying and constant when I was taking pictures